pkg/sfu/receiver.go

```diff
@@ -192,11 +192,7 @@ func NewWebRTCReceiver(
 	}
 
 	w.streamTrackerManager = NewStreamTrackerManager(logger, trackInfo, w.isSVC, w.codec.ClockRate, trackersConfig)
-	w.streamTrackerManager.OnAvailableLayersChanged(w.downTrackLayerChange)
-	w.streamTrackerManager.OnBitrateAvailabilityChanged(w.downTrackBitrateAvailabilityChange)
-	w.streamTrackerManager.OnMaxPublishedLayerChanged(w.downTrackMaxPublishedLayerChange)
-	w.streamTrackerManager.OnMaxTemporalLayerSeenChanged(w.downTrackMaxTemporalLayerSeenChange)
-	w.streamTrackerManager.OnBitrateReport(w.downTrackBitrateReport)
+	w.streamTrackerManager.SetListener(w)
```

Comment:
Using single listener for now.

Also, there is another place like this in `StreamAllocator` which I will adress in a subsequent PR.